client/js/components/user/DpOrganisationList/DpOrganisationFormFields.vue

```diff
@@ -309,6 +297,18 @@
             </div>
           </div>
         </template>
+
+        <dp-checkbox
+          v-if="hasPermission('feature_manage_procedure_creation_permission')"
+          id="procedureCreatePermission"
```

Comment:
this needs an index or something, that it is unique in a list

Reply:
indeed.. a5ab31e3e9d5916b94ec3c5c4a73b63e43f1962b